UI-ARCHITECTURE.md

```
@@ -0,0 +1,399 @@
+        <ToolbarItem><Button>Create fleet</Button></ToolbarItem>
+      </ToolbarContent>
+    </Toolbar>
+    <Table columns={columns} ...>
+      <Tbody>
+        {items.map(item => <FleetRow key={...} fleet={item} />)}
+      </Tbody>
+    </Table>
+    <TablePagination pagination={pagination} />
+  </ListPageBody>
+</ListPage>
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)